siddhi-test-framework/src/test/java/io/siddhi/distribution/test/framework/TemperatureAlertAppTest.java

```diff
@@ -200,7 +200,6 @@ public void testDBPersistence() throws SQLException, InterruptedException, IOExc
                 "}");
         ResultSet resultSet = null;
         try {
-            Thread.sleep(1000);
```

Comment:
Check whether removing thread.sleep works. Because, we need wait for few seconds before querying the DB.

Reply:
Works without this.But do I need to add it?